tools/Customization/DevHome.Customization/ViewModels/GeneralSystemViewModel.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.WinUI;
+using CommunityToolkit.WinUI.Behaviors;
+using DevHome.Common.Extensions;
+using DevHome.Common.Helpers;
+using DevHome.Common.Models;
+using DevHome.Common.Scripts;
+using DevHome.Common.Services;
+using DevHome.Customization.TelemetryEvents;
+using Microsoft.Internal.Windows.DevHome.Helpers;
+using Microsoft.UI.Dispatching;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Win32;
+using Serilog;
+
+namespace DevHome.Customization.ViewModels;
+
+public partial class GeneralSystemViewModel : ObservableObject
+{
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(GeneralSystemViewModel));
+
+    private readonly DispatcherQueue _dispatcherQueue;
+
+    private readonly StringResource _stringResource;
+
+    private readonly ShellSettings _shellSettings;
+
+    private readonly bool _isUserAdministrator = WindowsIdentityHelper.IsUserAdministrator();
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs { get; }
+
+    private StackedNotificationsBehavior? _notificationQueue;
+
+    private AsyncRelayCommand<bool> ModifyLongPathsCommand { get; }
+
+    public bool CanModifyLongPaths => _isUserAdministrator && !ModifyLongPathsCommand.IsRunning;
+
+    public GeneralSystemViewModel(DispatcherQueue dispatcherQueue)
+    {
+        _dispatcherQueue = dispatcherQueue;
+        _shellSettings = new ShellSettings();
+
+        _stringResource = new StringResource("DevHome.Customization.pri", "DevHome.Customization/Resources");
+        Breadcrumbs =
+        [
+            new(_stringResource.GetLocalized("MainPage_Header"), typeof(MainPageViewModel).FullName!),
+            new(_stringResource.GetLocalized("GeneralSystem_Header"), typeof(GeneralSystemViewModel).FullName!)
+        ];
+
+        ModifyLongPathsCommand = new AsyncRelayCommand<bool>(ModifyLongPathsAsync);
+        ModifyLongPathsCommand.PropertyChanged += async (s, e) =>
+        {
+            if (e.PropertyName == nameof(ModifyLongPathsCommand.IsRunning))
+            {
+                await _dispatcherQueue.EnqueueAsync(() => OnPropertyChanged(nameof(CanModifyLongPaths)));
+            }
+        };
+    }
+
+    public void Initialize(StackedNotificationsBehavior notificationQueue)
+    {
+        _notificationQueue = notificationQueue;
+    }
+
+    public void Uninitialize()
+    {
+        _notificationQueue = null;
+    }
+
+    public bool EndTaskOnTaskBarEnabled

```

Comment:
NIT: Add `Is` to this and `LongPathEnabled` resulting in `IsEndTaskOnTaskBarEnabled`